tests/net/buf_simple/prj.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+CONFIG_ASSERT_VERBOSE=y
+CONFIG_ASSERT_ON_ERRORS=y
+CONFIG_NET_BUF=y
+#CONFIG_NET_BUF_LOG=y
+#CONFIG_NET_BUF_LOG_LEVEL_DBG=y
+#CONFIG_NET_BUF_SIMPLE_LOG=y
```

Comment:
Should these be removed, or uncommented?

Reply:
left as it is

Reply:
Why?